website_sale_hide_empty_category/readme/USAGE.md

```diff
@@ -0,0 +1,2 @@
+Simply install the module and refresh your view. You will see the empty
+categories disappear
```

Comment:
It should be mentioned that the behavior can be configured with the editor

Reply:
Done, thanks!